Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.fedoraproject.org/fedora:30
+
+ENV LANG=en_US.UTF-8 \
+    PYTHONDONTWRITEBYTECODE=yes \
+    HOME=/tmp/packit-generator
+
+COPY requirements.txt requirements.sh files/packit-run.sh /tmp/packit-generator/
```

Comment:
why copy the files in when they are not used?

Reply:
We do, except `requirements.txt` which solved my suggestion.